### DISCUSSION TITLE
Allow to use it with .dev domains

### DISCUSSION THREAD

**Discussion by adambrangenberg** [nextcloud/all-in-one#778] [Ideas] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Setup AIO
2. Put any .dev domain into the domain field
3. Struggle because you're required to put in a domain

### Expected behavior 
It should accept 

### Actual behavior 
It doesn't. I think it's because in the logs it requests with http protocol which .dev domains don't allow

### Host OS 
Raspberry Pi OS Lite

#### Nextcloud AIO version 
1.3.1

**Comment by szaimen** [comment]
What are dev domains?

**Comment by ghost** [comment]
> What are dev domains?

I think domains with the Top-Level-Domain ending .dev

**Comment by szaimen** [comment]
I'll move this to discussions because it is a feature request if dev domains do not accept http connections.

**Reply by szaimen** [reply]
However you may be able to use it behind a reverse proxy. The domaincheck there is done over https...

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

**Reply by adambrangenberg** [reply]
Thank you, I'll try it tomorrow.

**Reply by adambrangenberg** [reply]
Hey, I've been following the guide (also optional steps), and it still won't work. In AIO Interface it says:
DNS config is not set for this domain or the domain is not a valid domain! (It was found to be set to 'nextcloud.tigerbyte.dev')

My DNS under nextcloud is an A-redirect (iP v4) to the PIs IP-Adress.

**Reply by szaimen** [reply]
So is this dns entry only set locally?

**Reply by adambrangenberg** [reply]
I've just put the IP of the Pi into my Nitrado dashboard 🤷‍♂️

**Reply by szaimen** [reply]
So is the domain and ip-address publicly set/reachable?

**Comment by szaimen** [comment]
Any update here?

**Reply by adambrangenberg** [reply]
Nothing worked, just used another app :x

**Reply by szaimen** [reply]
I see. Since 1.5.0 it is now possible to skip the domain validation. This may have helped setting it up.

**Comment by szaimen** [comment, marked as answer]
I hope this is resolved by disabling the domain validation which is now possible since 1.5.0
